packageJsonAnalyzer/codemodItem.ts

```diff
@@ -0,0 +1,31 @@
+import { TreeItem, TreeItemCollapsibleState } from 'vscode';
+import path from 'path';
+import { buildCodemodItemHash } from './utils';
+import { CodemodHash } from './types';
+
+export class CodemodItem extends TreeItem {
```

Comment:
I think this should be a separate POJSO type (like ElementItem) and there should be a builder that maps that type into a tree item.

The rationale is that here the new derived class just overrides that properties of `TreeItem`, there are no new specific methods to that class.